content/docs/fairing/gcp-kubeflow-notebook.md

```diff
@@ -0,0 +1,47 @@
++++
+title = "Train and Deploy on GCP from a Kubeflow Notebook"
+description = "Use Kubeflow Fairing to train and deploy a model on Google Cloud Platform (GCP) from a notebook that is hosted on Kubeflow."
+weight = 35
++++
+
+This guide introduces you to using Kubeflow Fairing to train and deploy a
+model to Kubeflow on Google Kubernetes Engine (GKE), and Google Cloud ML Engine.
+As an example, this guide uses a notebook that is hosted on Kubeflow to
+demonstrate how to:
+
+*  Train an XGBoost model in a notebook,
+*  Use Kubeflow Fairing to train an XGBoost model remotely on Kubeflow,
+*  Use Kubeflow Fairing to train an XGBoost model remotely on Cloud ML Engine, 
+*  Use Kubeflow Fairing to deploy a trained model to Kubeflow, and
+*  Call the deployed endpoint for predictions.
+
+## Use Kubeflow Fairing to train a model in a notebook, on Kubeflow, and on GCP
+
+1.  Use the Kubeflow user interface to open your hosted notebook environment
```

Comment:
We should probably have an introductory sentence/phrase to set the stage for this list.

Reply:
I've removed the second-level heading to make the page introduction the introduction for this section as well. PTAL

Reply:
I think we need an intro sentence. It looks a little weird just starting a numbered list.